indigo/distribution.yaml

```diff
@@ -9137,16 +9137,16 @@ repositories:
     doc:
       type: git
       url: https://github.com/baalexander/rospy_message_converter.git
-      version: indigo-devel
+      version: master
     release:
       tags:
         release: release/indigo/{package}/{version}
-      url: https://github.com/jihoonl/rospy_message_converter-release.git
-      version: 0.3.0-2
+      url: https://github.com/baalexander/rospy_message_converter-release.git
```

Comment:
Sorry, you should add `--override-release-repository-url https://github.com/baalexander/rospy_message_converter-release.git` to the bloom-release command as well.